CHANGELOG.rst

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+1.0.0 (2019-06-07)
```

Comment:
Is yours a breaking change that satisfies a major version jump?

I'd prefer to get more of the open issue fixed before releasing a version 1.0.

Reply:
I change this one to 0.12.0, do you agree ?

Reply:
Fine.